numba_dpex/core/kernel_interface/dispatcher.py

```diff
@@ -143,6 +147,28 @@ def __init__(
             self._has_specializations = False
             self._specialization_cache = NullCache()
 
+    def _create_func_hash(self):
+        """Creates a tuple of sha256 hashes out of code and variable bytes extracted from the compiled funtion."""
```

Comment:
Docstring is more the 70 chars. Please add line breaks.

Reply:
Done